docs/users/settings-appearance.md

```diff
@@ -42,7 +42,7 @@ You can customize the appearance of the website and app.
 <Tabs groupId="os">
   <TabItem value="web" label="Website">
 
-    The website always uses a data table orientated design for the day summary and timeline. 
+    The website always uses a data table orientated design for the logbook and timeline views. You can toggle between views after the **View as* text in the upper right-hand corner of the page.
```

Comment:
Highlighting the option to both view the geyser activity of a day as day summary and timeline only for the website is inconsistent, as that is also available for the apps (as outlined in view-date.md). I would suggest either removing the information or adding an own header and providing the information for all clients.